docs/recon_configurations/README.md

```diff
@@ -345,6 +345,44 @@ class Transformation:
 ```
 transformations=[Transformation(column_name)="array_col",source=sort_array_input(array_col),target=sort_array_input(array_col)]
 ```
+> **Note:** `NULL` values are defaulted to `_null_recon` in the transformation expression.If User is looking for specific behaviour, they should tweak the rules in the transformation expression accordingly in these files: 1. [expression_generator.py](https://github.com/databrickslabs/remorph/tree/main/src/databricks/labs/remorph/reconcile/query_builder/expression_generator.py) 2. [sampling_query.py](https://github.com/databrickslabs/remorph/tree/main/src/databricks/labs/remorph/reconcile/query_builder/sampling_query.py)
```

Comment:
change the language to override the rules through user transformation.

Reply:
Updated